src/models/user.js

```diff
@@ -19,6 +19,7 @@ export default (sequelize, DataTypes) => {
   // Associations
   User.associate = (models) => {
     User.belongsToMany(models.group, { through: models.groupSubscription })
+    User.hasMany(models.group, { as: 'myGroups', foreignKey: 'sellerId' })
```

Comment:
This can be simply "groups".

Reply:
@abelosorio But the relationship above is about groups, When I do user.groups this is returning me the subscrioption groups

Reply:
Resolved